lib/bankster/iban.ex

```diff
@@ -4,122 +4,124 @@ defmodule Bankster.Iban do
+    "MD" => %{length: 24, rule: ~r/^([0-9A-Z]{2})[0-9A-Z]{18}$/i},
+    "ME" => %{length: 22, rule: ~r/^([0-9]{3})[0-9]{15}$/i},
+    "MF" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "MG" => %{length: 27, rule: ~r/^[0-9]{23}$/i}, # No auto-BIC
+    "MK" => %{length: 19, rule: ~r/^([0-9]{3})[0-9A-Z]{10}[0-9]{2}$/i},
+    "ML" => %{length: 28, rule: ~r/^[A-Z]{1}[0-9]{23}$/i}, # No auto-BIC
+    "MQ" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "MR" => %{length: 27, rule: ~r/^([0-9]{5})([0-9]{5})[0-9]{13}$/i},
+    "MT" => %{length: 31, rule: ~r/^([A-Z]{4})([0-9]{5})[0-9A-Z]{18}$/i},
+    "MU" => %{length: 30, rule: ~r/^([A-Z]{4}[0-9]{2})([0-9]{2})[0-9]{15}[A-Z]{3}$/i},
+    "MZ" => %{length: 25, rule: ~r/^[0-9]{21}$/i}, # No auto-BIC
+    "NC" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "NE" => %{length: 28, rule: ~r/^[A-Z]{2}[0-9]{22}$/i}, # No auto-BIC
+    "NI" => %{length: 32, rule: ~r/^[A-Z]{4}[0-9]{24}$/i}, # No auto-BIC
+    "NL" => %{length: 18, rule: ~r/^([A-Z]{4})[0-9]{10}$/i},
+    "NO" => %{length: 15, rule: ~r/^([0-9]{4})[0-9]{7}$/i},
+    "PF" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "PK" => %{length: 24, rule: ~r/^([A-Z]{4})[0-9A-Z]{16}$/i},
+    "PL" => %{length: 28, rule: ~r/^([0-9]{3})([0-9]{4})[0-9]{17}$/i},
+    "PM" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "PS" => %{length: 29, rule: ~r/^([A-Z]{4})[0-9A-Z]{21}$/i},
+    "PT" => %{length: 25, rule: ~r/^([0-9]{4})([0-9]{4})[0-9]{13}$/i},
+    "QA" => %{length: 29, rule: ~r/^([A-Z]{4})[0-9]{4}[0-9A-Z]{17}$/i},
+    "RE" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "RO" => %{length: 24, rule: ~r/^([A-Z]{4})[0-9A-Z]{16}$/i},
+    "RS" => %{length: 22, rule: ~r/^([0-9]{3})[0-9]{15}$/i},
+    "SA" => %{length: 24, rule: ~r/^([0-9]{2})[0-9A-Z]{18}$/i},
+    "SC" => %{length: 31, rule: ~r/^([A-Z]{4}[0-9]{2})([0-9]{2})[0-9]{16}[A-Z]{3}$/i},
+    "SD" => %{length: 18, rule: ~r/^([0-9]{2})[0-9]{12}$/i},
+    "SE" => %{length: 24, rule: ~r/^([0-9]{3})[0-9]{17}$/i},
+    "SI" => %{length: 19, rule: ~r/^([0-9]{2})([0-9]{3})[0-9]{10}$/i},
+    "SK" => %{length: 24, rule: ~r/^([0-9]{4})[0-9]{16}$/i},
+    "SM" => %{length: 27, rule: ~r/^[A-Z]{1}([0-9]{5})([0-9]{5})[0-9A-Z]{12}$/i},
+    "SN" => %{length: 28, rule: ~r/^[A-Z]{1}[0-9]{23}$/i}, # No auto-BIC
+    "ST" => %{length: 25, rule: ~r/^([0-9]{4})([0-9]{4})[0-9]{13}$/i},
+    "SV" => %{length: 28, rule: ~r/^([0-9A-Z]{4})[0-9]{20}$/i},
+    "TD" => %{length: 27, rule: ~r/^[0-9]{23}$/i}, # No auto-BIC
+    "TF" => %{length: 27, rule: ~r/^[0-9]{10}[0-9A-Z]{11}[0-9]{2}$/i}, # No auto-BIC
+    "TG" => %{length: 28, rule: ~r/^[A-Z]{2}[0-9]{22}$/i}, # No auto-BIC
+    "TL" => %{length: 23, rule: ~r/^([0-9]{3})[0-9]{16}$/i},
+    "TN" => %{length: 24, rule: ~r/^([0-9]{2})([0-9]{3})[0-9]{13}$/i},
```

Comment:
According to the entry at https://de.wikipedia.org/wiki/Internationale_Bankkontonummer the regex should be `~r/^([0-9]{2})([0-9]{3})[0-9]{15}$/i`